crates/turbopack-ecmascript/src/analyzer/graph.rs

```diff
@@ -115,13 +115,15 @@ pub enum Effect {
         /// The ast path to the condition.
         ast_path: Vec<AstParentKind>,
         span: Span,
+        in_try: bool,
```

Comment:
I guess that doesn't need the flag, but nevermind...

Reply:
Unfortunately, it is actually used